packages/amplify-cli/src/config-steps/c0-analyzeProject.js

```diff
@@ -49,15 +49,15 @@ async function configureProjectName(context) {
 }
 
 function isProjectNameValid(projectName) {
-  return projectName && projectName.length >= 3 && projectName.length <= 20 && /[a-zA-Z0-9]/g.test(projectName);
```

Comment:
if we're going to use this reg ex in multiple places we should declare it as a const somewhere an import it when needed

Reply:
Lol, didn't read the other comments first, looks like this has already been pointed out